apkid/rules/apk/packers.yar

```diff
@@ -27,6 +27,25 @@
 
 include "common.yar"
 
+rule dexprotector : packer
+{
+ /**
+ * DexProtector v6.x.x Both Standard And Business Edition And Demo // Edited By Jasi2169
+ **/
+  meta:
+    description = "Dexprotector Licel"
```

Comment:
Could you change description to just "DexProtector"? That appears to be the product name.